app/controllers/registration/RegistrationController.scala

```diff
@@ -31,37 +31,33 @@ import utils.ControllerHelper._
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import models.enums._
+import services.AuditService
 import uk.gov.hmrc.http._
+import uk.gov.hmrc.play.audit.http.connector.AuditResult
 
 /**
- * Created by yasar on 2/5/15.
- */
+  * Created by yasar on 2/5/15.
+  */
 object RegistrationController extends RegistrationController {
   val desConnector = IhtConnector
+
   override def metrics: Metrics = Metrics
+
+  override def auditService = AuditService
```

Comment:
It looks like the unit tests won't compile because the auditService is not defined on the RegistrationControllerTest controller